src/window.rs

```diff
@@ -1129,9 +1129,9 @@ impl Window {
     ///
     /// - **macOS:** This is an app-wide setting.
     /// - **Wayland:** You can also use `WINIT_WAYLAND_CSD_THEME` env variable to set the theme.
-    /// Possible values for env variable are: "dark" and light". When unspecified, a theme is automatically selected.
-    /// -**x11:** Sets `_GTK_THEME_VARIANT` hint to `dark` or `light` and if `None` is used, it will default to  [`Theme::Dark`].
-    /// - **iOS / Android / Web / x11 / Orbital:** Unsupported.
+    ///   Possible values for env variable are: "dark" and light". When unspecified, a theme is automatically selected.
+    /// - **x11:** Sets `_GTK_THEME_VARIANT` hint to `dark` or `light` and if `None` is used, it will default to  [`Theme::Dark`].
```

Comment:
```suggestion
    /// - **X11:** Sets `_GTK_THEME_VARIANT` hint to `dark` or `light` and if `None` is used, it will default to  [`Theme::Dark`].
```

Just noticed that it was lower case....